patches/build.gradle.patch

```diff
@@ -0,0 +1,9 @@
+--- /tmp/tmp-status-mobile-215e516b2/tmp.iCGv95VzNg/build.gradle	2024-08-20 19:39:24.147334000 +0530
++++ ./node_modules/@react-native-community/blur/android/build.gradle	2024-08-20 19:39:27.720555147 +0530
+@@ -138,5 +138,5 @@
+   implementation "com.facebook.react:react-native:+"
+   // From node_modules
+ 
+-  implementation 'com.github.Dimezis:BlurView:version-2.0.3'
++  implementation 'com.github.Dimezis:BlurView:version-2.0.4'
```

Comment:
This will not work, as the version of Node modules we use in `gradle` Nix shell is not patched:
https://github.com/status-im/status-mobile/blob/7b34a4fb1b03a1533dfa974eb4d1c3e66470d3f9/nix/shells.nix#L53-L58

Reply:
Good catch, I've removed this block and it seems like the patched deps are fetched now.

Reply:
One possibility is to split patching of `nodejs` derivation into two stages: `nodejs-patched` and `nodejs-offline`. That way we can use the patched version without using the version that modifies Gradle configs to not reference remote repositories.

Though at the same time, I kinda wonder if that second type of patching is still necessary.